webview/src/plots/components/ZoomablePlot.tsx

```diff
@@ -30,6 +31,7 @@ export const ZoomablePlot: React.FC<ZoomablePlotProps> = ({
   const { data, content: spec } = useGetPlot(section, id, createdSpec)
   const dispatch = useDispatch()
   const currentPlotProps = useRef<VegaLiteProps>()
+  const onPlotViewReady = useSetupSmoothPlot(id)
```

Comment:
[C] I think this could have a more descriptive name.

Reply:
[Q] Also, would it be a better idea to pass this in for smooth plots only instead of calling this for all zoomable plots?  Unless I have missed something custom plots should never need to call `useSetupSmoothPlot` and having the hook floating around for all plots makes things harder to follow.